lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view_web.dart

```diff
@@ -646,6 +672,24 @@ class MailboxDashBoardView extends BaseMailboxDashBoardView {
       .toList();
   }
 
+  List<PopupMenuEntry> popupMenuEmailSortOrderType(
+    BuildContext context,
+    EmailSortOrderType? sortOrderSelected,
+    {Function(EmailSortOrderType)? onCallBack}
+  ) {
+    return EmailSortOrderType.values
+      .map((sortType) => PopupMenuItem(
+        padding: EdgeInsets.zero,
+        child: PopupItemNoIconWidget(
+          sortType.getTitle(context),
+          svgIconSelected: imagePaths.icFilterSelected,
+          maxWidth: 332,
```

Comment:
a magic number. Is it any constrain?

Reply:
The reason for this number is because I want to keep the width of the popup menu the same when selecting different items, and as long as the titles of the items are constant, I don't think it has any constraints!